modules/end-user-guide/partials/proc_logging-in-to-che-server-using-cli.adoc

```diff
@@ -9,13 +9,11 @@ This section describes how to log in to {prod-short} using {prod-cli} tool by co
 * {prod} Dashboard is opened in a browser.
 
 .Procedure
-. Open the user's context menu in the lower-left corner and select `Copy Login Command`
-+
-image::loggingin/copy_login_command_menu.png[copy_login_command_menu.png]
+. Open the user's context menu in the top-right corner
```

Comment:
For reviewers to easier identify if top-right corner is good enough description for it:

![Screenshot_20201230_201402](https://user-images.githubusercontent.com/5887312/103372863-e5e1b600-4adb-11eb-887a-811161793751.png)